core/lib/server/pub.db.test.ts

```diff
@@ -85,6 +86,7 @@ describe("createPubRecursive", () => {
 					[`${community.slug}:title`]: "test title",
 				},
 			},
+			lastModifiedBy: createLastModifiedBy("system"),
```

Comment:
having to add this everywhere in tests is annoying, but it's better than forgetting to put it there when you should! this is why is chose to not include a default for `lastModifiedBy`: we should always think about who's modifying the content.